utils/manifest-utils/src/keystore/x509_keystore.rs

```diff
@@ -645,69 +506,6 @@ fn cert_subject_entries(cert: &X509Ref, nid: Nid) -> Option<String> {
     Some(entries)
 }
 
-#[derive(Clone, Default)]
-pub struct PermissionsManager {
-    permissions: HashMap<String, Vec<CertPermissions>>,
-}
-
-impl PermissionsManager {
-    pub fn load(cert_dir: impl AsRef<Path>) -> anyhow::Result<PermissionsManager> {
-        let path = cert_dir.as_ref().join(PERMISSIONS_FILE);
-        let content = match std::fs::read_to_string(path) {
-            Ok(content) if !content.is_empty() => content,
-            _ => return Ok(Default::default()),
-        };
-        let permissions = serde_json::from_str(&content)?;
-        Ok(PermissionsManager { permissions })
-    }
-
-    pub fn set(&mut self, cert: &str, mut permissions: Vec<CertPermissions>) {
-        if permissions.contains(&CertPermissions::All) {
-            let supports_unverified_permissions =
-                permissions.contains(&CertPermissions::UnverifiedPermissionsChain);
-            permissions.clear();
-            permissions.push(CertPermissions::All);
-            if supports_unverified_permissions {
-                permissions.push(CertPermissions::UnverifiedPermissionsChain);
-            }
-        }
-        self.permissions.insert(cert.to_string(), permissions);
-    }
-
-    pub fn set_x509(&mut self, id: &str, permissions: Vec<CertPermissions>) {
-        self.set(id, permissions)
-    }
-
-    pub fn set_many(
-        &mut self,
-        cert_ids: &Vec<&str>,
-        leaf_cert_ids: &Vec<&str>,
-        permissions: &[CertPermissions],
-        whole_chain: bool,
-    ) {
-        let permissions = Vec::from(permissions);
-        let cert_ids = if whole_chain { cert_ids } else { leaf_cert_ids };
-        for id in cert_ids {
-            self.set_x509(id, permissions.clone());
-        }
-    }
-
-    /// If we don't have this certificate registered, it means it has no permissions,
-    /// so empty vector is returned.
-    pub fn get(&self, cert: &X509Ref) -> Vec<CertPermissions> {
-        let id = match cert_to_id(cert) {
-            Ok(id) => id,
-            Err(_) => return vec![],
-        };
-        self.permissions.get(&id).cloned().unwrap_or_default()
-    }
-
-    pub fn save(&self, path: &Path) -> anyhow::Result<()> {
-        let mut file = File::create(path.join(PERMISSIONS_FILE))?;
```

Comment:
`cert-permissions.json` file is no longer created so we no longer need `ya-provider pre-install` logic applying permissions default values https://github.com/golemfactory/yagna/blob/remove-permissions-logic/agent/provider/src/cli/pre_install.rs#L6

Reply:
Also maybe we should delete this file using `ya-installer` command?
https://github.com/golemfactory/ya-installer/blob/master/commands.json